agent/engine/ordering_integ_test.go

```diff
@@ -161,6 +161,11 @@ func TestDependencyComplete(t *testing.T) {
 // Container 'parent' depends on  container 'dependency' to START. We ensure that the 'parent' container starts only
 // after the 'dependency' container has started.
 func TestDependencyStart(t *testing.T) {
+	// Skip these tests on WS 2016 until the failures are root-caused.
```

Comment:
Unrelated to this change, but the rest of this tests here are already skipped on WS 2016, adding this one because it is failing intermittently.